charter.md

```diff
@@ -53,15 +53,164 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+In the instance that a general election is called as a result of the
+[dissolution](#dissolution) of the current committee, the following rules will
+apply:
+
+* members of the current committee will not be allowed to run in the new
+  election
+* a new committee will be bootstrapped with:
+  * the top four candidates serving a two-year term
+  * the next three candidates serving a one-year team
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this process fails to fill the seat, a special election for that position will
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
+
+If committee members believe that the committee is no longer able to act in
+accordance with the above [mission](#mission), the committee may dissolve by a
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Dissolution by vote
+
+The committee must specify a date of dissolution. Dissolution requires an
+affirmative vote of 75% of committee members, and must be unanimous when the
+committee has four (4) or fewer members. If the committee is dissolved, all
+seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+In the event that 50% (or more) of the committee's members
+[resign](#resignation) at the same time, the entire committee will be disbanded.
```

Comment:
I think it's critical in both of these sections to specify if the outgoing members can run in the subsequent election or not. Making that different for the two mechanisms here might give a reason for there being two mechanisms.

Reply:
Need to be clear that this is 50% "rounded up", e.g. 4 out of 7